### PR TITLE
Minor corrections on documentation

### PR DESCRIPTION
**Pull request by davidefalcone1**
# Description

Fixed command used to turn off authentication.
Fixed table in scenarios.

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: docs/pages/User/Scenarios/_index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ On-premise cluster (K3s or K8s) exposed through the Internet peered with a Manag
 On-premise cluster (K3s or K8s) peered with another on-premise cluster (K3s or K8s) in the same LAN.
 From the discovery perspective, if the clusters you would like to connect are in the same L2 broadcast domain, the Liqo discovery mechanism based on mDNS will handle the discovery automatically. If you have your clusters in different L3 domains, you have to manually [create](/user/post-install/discovery#forging-the-foreigncluster) a *foreign_cluster* resource or rely on [DNS discovery](/user/post-install/discovery#manual-configuration).
 
-|  | Cluster A (On-prem behind NAT) | Cluster B (Cloud) |
+|  | Cluster A (On-prem behind NAT) | Cluster B (On-prem behind NAT) |
```
Comment (palexster) [inline docs/pages/User/Scenarios/_index.md:52]: I think in this line they are both just On-Prem (No nat). Also, the one you did not modify is incorrect (Cluster A).

Comment (davidefalcone1) [reply docs/pages/User/Scenarios/_index.md]: Ok, it should be fixed now.

**Review by mlavacca — APPROVED** [review]

**Comment by mlavacca** [comment]
Hi @davidefalcone1! Thanks for your first PR!

**Pull request merged**